modules/auxiliary/scanner/rdp/cve_2019_0708_bluekeep.rb

```diff
@@ -90,9 +81,8 @@ def rdp_reachable
     return false
   end
 
-  def check_host(ip)
+  def check_host(_ip)
```

Comment:
are you sure this should change to _ip?

Reply:
It's not directly called anywhere in that function and appears to function correctly.  It was changed to `_ip` based on feedback from `rubocop` about its lack of use.

Reply:
alright

Reply:
Did you rebase? I can't find my comment explaining the suggestion.

Reply:
It's settled. :)